test/functional/rpc_mempool_info.py

```diff
@@ -78,22 +78,23 @@ def create_tx(**kwargs):
         assert_equal(result, [ {'txid' : txidB, 'vout' : 0, 'spendingtxid' : txidD}, {'txid' : txidG, 'vout' : 3} ])
 
         self.log.info("Unknown input fields")
-        assert_raises_rpc_error(-3, "Unexpected key unknown", self.nodes[0].gettxspendingprevout, [{'txid' : txidC, 'vout' : 1, 'unknown' : 42}])
+        assert_raises_rpc_error(-3, "JSON value of type string is not of expected type number", self.nodes[0].gettxspendingprevout, [{'txid' : txidC, 'vout' : 1, 'unknown' : 42}])
 
         self.log.info("Invalid vout provided")
         assert_raises_rpc_error(-8, "Invalid parameter, vout cannot be negative", self.nodes[0].gettxspendingprevout, [{'txid' : txidA, 'vout' : -1}])
 
         self.log.info("Invalid txid provided")
-        assert_raises_rpc_error(-3, "Expected type string for txid, got number", self.nodes[0].gettxspendingprevout, [{'txid' : 42, 'vout' : 0}])
+
```

Comment:
why a newline?

Reply:
Accidentally committed. I will correct it in the next review commit.